sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -300,6 +300,11 @@ functionCall
     | aggregateFunction                                             #aggregateFunctionCall
     | aggregateFunction (orderByClause)? filterClause               #filteredAggregationFunctionCall
     | relevanceFunction                                             #relevanceFunctionCall
+    | highlightFunction                                             #highlightFunctionCall
+    ;
+
+highlightFunction
+    : HIGHLIGHT LR_BRACKET relevanceField RR_BRACKET
```

Comment:
This allows to call highlight in WHERE clause too. Do we need this?

Reply:
I don't think it is needed in the WHERE clause.

Reply:
Looks like many expressions can be accepted in a WHERE clause. This will need to be addressed in another ticket.

Reply:
"it's a feature!"
If we really can't resolve that now, we can raise a follow-up bug.